force-app/main/default/lwc/eventManager/eventManager.js

```diff
@@ -0,0 +1,77 @@
+import { LightningElement, api } from 'lwc';
+import { subscribe}  from 'lightning/empApi';
+import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+
+export default class EventManager extends LightningElement {
+
+    cancelSubscription = {};
+    renewSubscription = {};
+     @api cancelChannel = '/event/AssetCancelInitiatedEvent';
+     @api renewChannel = '/event/AssetRenewInitiatedEvent';
+
+    // Initializes the component
+    connectedCallback() {
+        // Register listener
+        this.handleRenewSubscribe();
+        this.handleCancelSubscribe();
+    }
+
+    handleCancelSubscribe() {
+        // Callback invoked whenever a new event message is received
+        const thisReference = this;
+        const messageCallback = function(response) {
+            let obj = JSON.parse(JSON.stringify(response));
+            let message = 'Order created: ' + obj.data.payload.CancellationRecordId;
```

Comment:
You need to match the requestIdentifier before proceeding - otherwise you might be processing events that are not related to your UI.